Jenkinsfile

```diff
@@ -69,11 +79,43 @@ pipeline {
                             if (ecrLogin(TF_STATE_BUCKET_REGION) != 0 )  { error("Docker login to ECR failed") }
                             String dockerPushCommand = "docker push ${DOCKER_IMAGE}"
                             if (sh (label: "Pushing image", script: dockerPushCommand, returnStatus: true) !=0) { error("Docker push gpc-consumer image failed") }
+                            
+                            String dockerPushCommandWiremock = "docker push ${WIREMOCK_DOCKER_IMAGE}"
+                            if (publishWiremockImage) {
+                                if (sh (label: "Pushing Wiremock image", script: dockerPushCommandWiremock, returnStatus: true) !=0) { error("Docker push gpc-consumer-wiremock image failed") }
+                            }
                         }
                     }
                 }
             }
         }
+    
+        // stage('Deploy & Test') {
```

Comment:
as we are not going to deploy gpc with this pipeline, this section is not needed and should be removed